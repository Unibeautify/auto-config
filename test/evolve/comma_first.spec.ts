import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("comma_first", {
  beautifiers: [prettier, jsBeautify],
  language: "JavaScript",
  originalText: `const a = "a", b = "b", c = "c";`,
  desiredText: `const a = "a"\n  , b = "b"\n  , c = "c";`,
});
