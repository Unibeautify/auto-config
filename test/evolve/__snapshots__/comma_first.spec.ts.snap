// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comma_first: beautifiedText 1`] = `
const a = "a"
  , b = "b"
  , c = "c";
`;

exports[`comma_first: entity 1`] = `
Object {
  "options": Object {
    "beautifiers": Array [
      "JS-Beautify",
    ],
    "comma_first": true,
    "indent_size": 2,
  },
}
`;

exports[`comma_first: fitness 1`] = `1`;
